### DISCUSSION TITLE
How can I tell which formulae are dependancies and which aren't?

### DISCUSSION THREAD

**Discussion by Southpaw1496** [Homebrew/discussions#115] [Everyday usage] [answered]
I'm having a clear out on my Mac and I noticed that I have a grand total of 78 formulae, and to my knowledge I only installed three or four. But I know that a lot of formulae have dependancies, so is there any way to list only formulae that are not a dependent of another formulae, so that I don't break something in the process of my cleaning?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Manpage#deps-options-formula

**Comment by gromgit** [comment, marked as answer]
> I'm having a clear out on my Mac and I noticed that I have a grand total of 78 formulae, and to my knowledge I only installed three or four. But I know that a lot of formulae have dependancies, so is there any way to list only formulae that are not a dependent of another formulae, so that I don't break something in the process of my cleaning?

`brew leaves`: https://docs.brew.sh/Manpage#leaves

**Comment by jonchang** [comment]
The new `brew autoremove` command will be coming soon once we cut a new Homebrew release, which should help you clean up dependencies that you no longer need.
